### ISSUE TITLE
Exposing names in color_names.inc

### ISSUE THREAD

**Issue by willt** [godotengine/godot#11164]
This file is referenced in ColorN in the @GDScript documentation.
I'm looking for thoughts on a good way to pull this into the documentation because telling the user to go look at an include file in the source code is kind of mean. :)

https://github.com/godotengine/godot/blob/master/core/color_names.inc

**Comment by bojidar-bg** [comment]
Well, I guess we can just run a regex over it? Something like this should work:
```js
/^\s+_named_colors.insert("(\w+)", Color([0-9\.]+, [0-9\.]+, [0-9\.]+));$/
```

**Comment by willt** [comment]
Should we manually parse the file and add it to the docs or update the build system to do it automagically ? It would be trivial to write something in perl or python to parse and spit out the correct formatting to add in.

**Comment by akien-mga** [comment]
Documenting it manually once is enough, I don't think it's worth the trouble to have the buildsystem modify XML documentation for a list of colors that should not change (colors don't change much over time :P).

**Comment by poke1024** [comment]
Also see https://github.com/godotengine/godot/pull/14704 (at the end)

**Comment by akien-mga** [comment]
Fixed by #15455.
